src/base.jl

```diff
@@ -1075,33 +1126,35 @@ function check_bus(sys::System, bus::Bus, component::Component)
     end
 end
 
-function IS.compare_values(x::System, y::System)::Bool
-    match = true
-
-    if !IS.compare_values(x.data, y.data)
-        @debug "SystemData values do not match"
-        match = false
-    end
+function IS.compare_values(x::System, y::System)
 
     if x.basepower != y.basepower
         @debug "basepower does not match" x.basepower y.basepower
-        match = false
+        return false
```

Comment:
I wrote the original logic so that you would get error prints for both basepower mismatches and data mismatches.  I still prefer that.  That pattern exists in several places in the code.  Do you disagree with that pattern?

I suggest not changing logic in a "style fixes" PR.

Reply:
Resolved in https://github.com/NREL/PowerSystems.jl/pull/390/commits/03635a151b5c9a9580a476a0cc169029c8379ba2.